benchmarks/distributed/rpc/rl/agent.py

```diff
@@ -0,0 +1,96 @@
+import numpy as np
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+
+import torch.distributed.rpc as rpc
+from torch.distributed.rpc import RRef, remote
+from torch.distributions import Categorical
+
+from Observer import ObserverBase
+
+OBSERVER_NAME = "observer{}"
+
+gamma = 0.99
+
+
+class Policy(nn.Module):
+    def __init__(self, n_layers=0, in_features=10, out_features=10):
+        super(Policy, self).__init__()
+        self.n_layers = n_layers
+        self.in_features = in_features
+        self.out_features = out_features
+
+        self.model = nn.Sequential(
+            *[nn.Linear(in_features, out_features) for _ in range(n_layers + 1)])
```

Comment:
IIUC, if `in_features != out_features`, these layers cannot be stacked together. Maybe you can hardcode in_features and out_features in the first version? Say `in_features = out_features = 100`?